test/spec/SearchResults/initialization.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var SearchParameters = require('../../../src/SearchParameters');
+var SearchResults = require('../../../src/SearchResults');
+
+test('processingTime should be the sum of all individual times', function() {
```

Comment:
Is it sure that those queries are perform in sequence and not in parallel?

Reply:
I'm not sure what you mean

Reply:
It's not related to your PR but more on the global idea to sum up the time. It makes sense to do a sum only if the queries are triggered in sequence. But are we sure of that? Worth asking we won't change the implementation but would be interesting to know.

Reply:
Good point, we can ask about that